sgl-kernel/csrc/cpu/gemm.h

```diff
@@ -177,6 +177,15 @@ void tinygemm_kernel(
     bool brg,
     int64_t block_size_K);
 
+at::Tensor fp8_scaled_mm_cpu(
+    at::Tensor& mat1,
+    at::Tensor& mat2,
+    at::Tensor& scales2,
+    std::vector<int64_t> block_size,
+    std::optional<at::Tensor>& bias,
+    at::ScalarType out_dtype,
+    bool is_vnni);
+
```

Comment:
remove this one and use `extern` as here: https://github.com/mingfeima/sglang/blob/cpu_opt_ww11/sgl-kernel/csrc/cpu/qkv_proj.cpp#L326

Reply:
Thanks. Done.